src/extension/extensionServer.ts

```diff
@@ -123,6 +124,21 @@ export class ExtensionServer implements vscode.Disposable {
         return Q.resolve<void>(void 0);
     }
 
+    /**
+     * Message handler for OPEN_FILE_AT_LOCATION
+     */
+    private openFileAtLocation(filename: string, lineNumber: number): Q.Promise<any> {
+        // This should work when https://github.com/Microsoft/vscode/issues/2933 gets fixed
+        vscode.commands.executeCommand("vscode.openFolder", vscode.Uri.file(filename)).then(() => {
```

Comment:
If possible, you should convert this into a Q promise and return it. If you are not returning a promise, then it doesn't make sense to return Q.Promise<any>